### PR TITLE
Nise Generating Data in BigQ format

### PR DESCRIPTION
**Pull request by lrlevine6**
This issue adds changes to the current gcp nise implementation. Now the report generated will resemble that created by the BigQuery. Changes were made to the report generator as well as to gcp generators. This changes focus on the Compute Engine Generator. Changes to the CLI will still need to be adjusted. In addition, nise can generate daily reports when using the argument `--daily-report`.

### CODE REVIEW

File: nise/generators/gcp/gcp_generator.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 from nise.generators.generator import AbstractGenerator
 
-
+# Old report columns
```
Comment (myersCody) [inline nise/generators/gcp/gcp_generator.py:8]: I still need to delete the old columns then rename the `BIGQ_REPORT_COLUMNS` to `GCP_REPORT_COLUMNS`.

File: nise/generators/gcp/cloud_storage_generator.py [hunk 2]
```diff
@@ -28,26 +27,13 @@ class CloudStorageGenerator(GCPGenerator):
...
-        else:
-            storage = choice(self.STORAGE)
-            row["Line Item"] = storage[0]
-            row["Measurement1"] = storage[0]
-            row["Measurement1 Total Consumption"] = self.fake.pyint()
-            row["Measurement1 Units"] = storage[1]
-            row["Cost"] = self.fake.pyint()
-            row["Currency"] = "USD"
-            row["Description"] = storage[2]
+        # if self.attributes:
```
Comment (myersCody) [inline nise/generators/gcp/cloud_storage_generator.py:30]: I think we should delete these comments and leave a todo comment saying we need to complete the storage generator. Probably need to create a jira issue too for the nise storage generator.

File: nise/generators/gcp/project_generator.py [hunk 3]
```diff
@@ -1,30 +1,44 @@
...
+from random import choice
+
 from faker import Faker
 
 
 class ProjectGenerator:
     """Generator for GCP Compute Engine data."""
 
+    PROJECT_INFO = (  # id -  name, labels, ancestry_numbers
+        ("doug-cost-test", "Doug-cost-test", "[]", ""),
```
Comment (myersCody) [inline nise/generators/gcp/project_generator.py:11]: I think we may want to rename the default project names here. We may be able to do something auto generated. 🤷‍♂️

File: nise/report.py [hunk 4]
```diff
@@ -69,6 +68,8 @@
 from nise.upload import upload_to_s3
 from nise.util import LOG
 
+# from nise.generators.gcp import CloudStorageGenerator (Functionality to be added)
```
Comment (myersCody) [inline nise/report.py:71]: I need to delete this.

File: nise/generators/gcp/gcp_generator.py (hunk 1 above)
Comment (cgoodfred) [reply nise/generators/gcp/gcp_generator.py]: addressed here: https://github.com/project-koku/nise/pull/314/commits/3e23ad4e34e94d146ab961688c6555c6203673ff

File: nise/generators/gcp/cloud_storage_generator.py (hunk 2 above)
Comment (cgoodfred) [reply nise/generators/gcp/cloud_storage_generator.py]: addressed here: https://github.com/project-koku/nise/pull/314/commits/3e23ad4e34e94d146ab961688c6555c6203673ff

File: nise/generators/gcp/project_generator.py (hunk 3 above)
Comment (cgoodfred) [reply nise/generators/gcp/project_generator.py]: addressed here: https://github.com/project-koku/nise/pull/314/commits/3e23ad4e34e94d146ab961688c6555c6203673ff

File: nise/report.py (hunk 4 above)
Comment (cgoodfred) [reply nise/report.py]: addressed here: https://github.com/project-koku/nise/pull/314/commits/3e23ad4e34e94d146ab961688c6555c6203673ff

**Review by myersCody — APPROVED** [review]

**Pull request merged**